unicore/ask/service/schema.py

```diff
@@ -30,3 +30,36 @@ class QuestionSchema(colander.MappingSchema):
         validator=validators.question_type_validator)
     multiple = colander.SchemaNode(colander.Boolean(), default=False)
     options = Options(validator=validators.options_validator, default=[])
+
+
+class QuestionOptionSchemaPut(colander.MappingSchema):
+    uuid = colander.SchemaNode(
+        colander.String(),
+        validator=validators.option_uuid_validator, missing=None)
+    title = colander.SchemaNode(
+        colander.String(),
+        validator=validators.option_title_validator, missing=None)
```

Comment:
No, but `put` should allow for partial updates. So if you just need to change a question to a multi-select, you can just submit a put with `{'multiple': True}`